web/Dockerfile.okteto

```diff
@@ -1,15 +1,19 @@
-FROM node:10
-EXPOSE 3000 9229
+FROM node:10 as dev
```

Comment:
ouch

Reply:
yeah, I wanted to upgrade the node version but it's going to require some retooling that could take some time.  I was slow for Vendor web but I think I know the steps.  I can try to get it done immediately after this though.